aws-predictions-tfl/src/main/java/com/amplifyframework/predictions/TFLitePredictionsEscapeHatch.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions;
+
+import androidx.annotation.NonNull;
+
+import org.tensorflow.lite.Interpreter;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * An escape hatch to give low-level access to Tensorflow interpreter.
+ */
+public final class TFLitePredictionsEscapeHatch {
```

Comment:
yea I did this because we would need to return multiple types of AWS clients in the AWS plugin. If we directly use a map instead of encapsulating it, then it would need to be `Map<String, AmazonWebServiceClient>`, which I think is pretty ugly.